Formula/portable-ruby@2.2.rb

```diff
@@ -6,6 +6,7 @@ class PortableRubyAT22 < PortableFormula
   url "https://cache.ruby-lang.org/pub/ruby/2.2/ruby-2.2.7.tar.gz"
   sha256 "374184c6c5bbc88fb7bad422368d4053a236fb6587f0eff76146dcba57f93da5"
 
+  depends_on "make" => :build if MacOS.version < :leopard
```

Comment:
If this is also needed for other formulae,  we could move `depends_on "make" => :build` to `PortableFormulae`